application/LicenseAppService_test.go

```diff
@@ -118,6 +140,49 @@ func TestSameOrgAndServiceAddedTwiceNotPossible(t *testing.T) {
 	assert.Equal(t, 20, len(assignable))
 }
 
+func TestSameOrgAndServiceAddedTwiceIsPossibleInNonStrictModeWithoutLicenseChange(t *testing.T) {
```

Comment:
but.. .doesn't this now violate the "done" criteria?! same org and service added twice should return an error.